tests/python_package_test/test_engine.py

```diff
@@ -747,6 +747,23 @@ def test_feature_name(self):
         gbm = lgb.train(params, lgb_train, num_boost_round=5, feature_name=feature_names_with_space)
         self.assertListEqual(feature_names, gbm.feature_name())
 
+    def test_feature_name_with_non_ascii(self):
+        X_train = np.random.normal(size=(100, 4))
+        y_train = np.random.random(100)
+        # This has non-ascii strings.
+        feature_names = [u'F_零', u'F_一', u'F_二', u'F_三']
+        params = {'verbose': -1}
+        lgb_train = lgb.Dataset(X_train, y_train)
+
+        gbm = lgb.train(params, lgb_train, num_boost_round=5, feature_name=feature_names)
+        self.assertListEqual(feature_names, gbm.feature_name())
+
+        # save model
+        gbm.save_model('model_feature_names.txt')
```

Comment:
Add this to gitignore?
https://github.com/microsoft/LightGBM/blob/ce08242567e1e37e1f947038a78ac671c98ed18e/.gitignore#L379-L382

Reply:
I changed model name to be saved.